tests/integration/cli/stop_test.py

```diff
@@ -0,0 +1,22 @@
+from ...testcases import DustyIntegrationTestCase
+from ...fixtures import busybox_single_app_bundle_fixture
+
+class TestStopCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestStopCLI, self).setUp()
+        busybox_single_app_bundle_fixture(num_bundles=1)
+        self.run_command('bundles activate busyboxa busyboxa')
+        self.run_command('up')
```

Comment:
You need a try-wrapped `stop` in your `tearDown` function in case your tests fail